functional_tests/lightning/train-gpu-ddp.yea

```diff
@@ -3,7 +3,7 @@ plugin:
   - wandb
 tag:
   shard: service
-  skip: True
+  skip: true
```

Comment:
soon... it will run in our gpu shard 🤩

Reply:
yea, I'm going to add a bunch of tests to our GPU shard tomorrow 🚗